Module.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+declare(strict_types=1);
+
+namespace LinkedDataSets;
+
+use BulkExport\Job\Export as JobExport;
+use LinkedDataSets\Application\Job\CatalogDumpJob;
+use LinkedDataSets\Domain\Job\CreateCatalogDumpJob;
+use Laminas\EventManager\Event;
+use Laminas\EventManager\SharedEventManagerInterface;
+use Laminas\ServiceManager\ServiceLocatorInterface;
+use Omeka\Api\Adapter\ItemAdapter;
+use Omeka\Api\Manager;
+use Omeka\Api\Representation\ItemRepresentation;
+use Omeka\Api\Representation\ResourceClassRepresentation;
+use Omeka\Api\Request;
+use Omeka\Job\Dispatcher;
+use Omeka\Module\AbstractModule;
+use Omeka\Module\Exception\ModuleCannotInstallException;
+use Omeka\Module\Module as DefaultModule;
+use Omeka\Stdlib\Message;
+use Omeka\Api\Exception\ValidationException;
+
+final class Module extends AbstractModule
+{
+    private ?Dispatcher $dispatcher = null;
+    private $api = null;
+    private const MODULE_DEPENDENCIES = [
+        ['name' => 'CustomVocab', 'version' => '1.7.1'],
+        ['name' => 'AdvancedResourceTemplate', 'version' => '3.4.4.17'],
+        ['name' => 'NumericDataTypes', 'version' => '1.11.0'],
+    ];
+
+    private const FOLDERS = [
+        ['path' => 'files/datacatalogs/'],
+        ['path' => 'files/datadumps/'],
+    ];
+
+    public function getConfig()
+    {
+        return include __DIR__ . '/config/module.config.php';
+    }
+
+    public function setServiceLocator(ServiceLocatorInterface $serviceLocator)
+    {
+        $this->serviceLocator = $serviceLocator;
+    }
+
+    public function install(ServiceLocatorInterface $serviceLocator)
+    {
+
+//        $this->checkPrerequisites($serviceLocator);
+//        $this->createFoldersIfTheyDontExist();
+//        $this->installSchemaOrgVocabulary($serviceLocator);
+//        $this->installCustomVocabularies();
+//        $this->installTemplates();
+    }
+
+    protected function checkPrerequisites(ServiceLocatorInterface $serviceLocator): void
+    {
+        /* @var DefaultModule $module */
+        foreach (self::MODULE_DEPENDENCIES as $moduleDependency) {
+            $module = $serviceLocator->get('Omeka\ModuleManager')
```

Comment:
As per my review point :point_up: 

Retrieving the Omeka\ModuleManager would preferably be done via a private getter. Example:

```
private function getModuleManager(): ModuleManager
{
    return $serviceLocator->get('Omeka\ModuleManager');
}
```